extension-config.json

```diff
@@ -0,0 +1,49 @@
+{
+  "version": "1.0.0",
+  "id": "@shopgate/change-header-colors",
+  "components": [
+    {
+      "id": "Subscribers",
+      "path": "frontend/subscriptions/index.js",
+      "type": "subscribers"
+    }
+  ],
+  "configuration": {
+    "headerHomePageBackgroundColor": {
```

Comment:
Sorry to comment on this again. Can we make it `headerHomeBackgroundColor`. Currently it's `headerHomePage` for this one, but next one is `headerHome` which is kind of confusing in my opinion ;)

Reply:
No need to be sorry. I corrected this.